projects/core/components/primitive-textfield/value-decoration/value-decoration.component.ts

```diff
@@ -50,13 +50,18 @@ export class TuiValueDecorationComponent {
     }
 
     get value(): string {
-        return this.textfield.value;
+        return this.textfield.value || '';
     }
 
     get filler(): string {
         const {focused, placeholder, exampleText, value, textfield} = this;
 
-        if (focused && placeholder && exampleText) {
+        const skipFiller =
+            (focused && placeholder && exampleText) ||
+            (textfield.size == 's' && !value.length) ||
```

Comment:
We should probably add linter on `==` instead of `===`

Reply:
https://github.com/Tinkoff/taiga-ui/pull/1744

Reply:
fixed